src/main/scala/com/cognite/spark/datasource/TimeSeriesRelation.scala

```diff
@@ -87,13 +91,42 @@ class TimeSeriesRelation(config: RelationConfig)(val sqlContext: SQLContext)
 
   private def postRows(rows: Seq[Row]): IO[Unit] = {
     val timeSeriesItems = rows.map(r => fromRow[PostTimeSeriesItem](r))
-    post(config.apiKey, baseTimeSeriesUrl(config.project), timeSeriesItems, maxRetries)
-      .map(item => {
+    putOr(config.apiKey, baseTimeSeriesUrl(config.project), timeSeriesItems, maxRetries) {
+      case Response(Right(body), StatusCodes.NotFound, _, _, _) =>
+        decode[Error[TimeSeriesNamesNotFound]](body) match {
+          case Right(conflict) => resolveConflict(timeSeriesItems, conflict.error)
+          case Left(error) => IO.raiseError(error)
+        }
+    }.flatTap { _ =>
+      IO {
```

Comment:
We should only increase this metric if time series were created, and not do so if they were just updated. Maybe that's still what happens, but I'd like us to be sure about this.